utilities/ifix-migration-toolkit/mvnw.cmd

```diff
@@ -0,0 +1,188 @@
+@REM ----------------------------------------------------------------------------
```

Comment:
Delete this

Reply:
minor change, to be taken up as part of code cleanup.